{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/67/comments/33578971.json"}, "html": {"href": "#!/osrf/srcsim/issues/67#comment-33578971"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/67.json"}}, "type": "issue", "id": 67, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Problem connecting to server during setup"}, "content": {"raw": "things appear to be working after the Friday-Tuesday outage", "markup": "markdown", "html": "<p>things appear to be working after the Friday-Tuesday outage</p>", "type": "rendered"}, "created_on": "2017-01-11T18:00:20.507426+00:00", "user": {"display_name": "Stephen Hart", "uuid": "{b97e8c2f-f4f4-40ff-959d-0747b9e8e333}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bb97e8c2f-f4f4-40ff-959d-0747b9e8e333%7D"}, "html": {"href": "https://bitbucket.org/%7Bb97e8c2f-f4f4-40ff-959d-0747b9e8e333%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/35abf681325d18e2a555f6eed0032ad1d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSH-1.png"}}, "nickname": "swhart115", "type": "user", "account_id": "557058:42f9a849-0546-40b7-ae09-00e83169458b"}, "updated_on": null, "type": "issue_comment", "id": 33578971}